pkg/ui/v1alpha3/const.go

```diff
@@ -0,0 +1,21 @@
+package v1alpha3
+
+import (
+	"github.com/kubeflow/katib/pkg/util/v1alpha3/env"
+	"strings"
+)
+
+const (
+	AvailableNameSpaceEnvName = "KATIB_UI_AVAILABLE_NS"
+	ClusterRoleKey            = "KATIB_UI_CLUSTER_ROLE"
+)
+
+var (
+	availableNameSpaces, hasClusterRole = func() ([]string, bool) {
+		ns := env.GetEnvOrDefault(AvailableNameSpaceEnvName, ClusterRoleKey)
+		if ns == ClusterRoleKey {
+			return []string{""}, true
```

Comment:
I'm not confident but do you mistake it for `[]string{}`?

And I think you can use `init()` function like:

```go
var (
    availableNameSpaces []string
    hasClusterRole      bool
)

func init() {
    ns := env.GetEnvOrDefault(AvailableNameSpaceEnvName, ClusterRoleKey)
    ...
}
```

Reply:
@c-bata Thanks for your suggestions!
1. It should be `[]string{""}` which means no namespace options(restriction). If `[]string{}`, it will use default name space defined by `controller.v1alpha3/consts.DefaultKatibNamespace`.
2. I have replaced with init function

Reply:
I see. Thank you for your explanation!